database/schema.sql

```diff
@@ -133,7 +133,8 @@ CREATE SEQUENCE loaders_id_seq START 1;
 CREATE TABLE loaders (
 	id            numeric NOT NULL DEFAULT nextval('loaders_id_seq'),
 	loadername    character varying(256) NOT NULL,
-	loaderkey     character varying(64) NOT NULL,
+	keyprefix     character varying(256) NOT NULL,
+	loaderkey     character varying(256) NOT NULL,
```

Comment:
here `loaderkey` is the hexadecimal of a salt+hash, correct?
as I mentioned on IRC, I think it may be cleaner to store salt and hash in their own column as bytes, without any encoding. We can always convert them on the flight using sql as needed.